proto/ate.proto

```diff
@@ -643,6 +643,13 @@ message Ipv4Header {
   uint32 dscp = 4;
   uint32 ecn = 5;
   uint32 checksum = 7;
+  uint32 header_length = 8;
+  UIntRange protocol = 9;
```

Comment:
By default, IxNetwork infers this behavior from the next header, correct? What is the use case for a test wanting to set it explicitly?

Reply:
These are the following files “header_length”,  “total_length” and “protocols” are auto-populated in IxNetwork.
protocols : As far as our understanding goes user wants to set IP Protocol value as 94 (IP-within-IP Encapsulation Protocol) for DUT testing.
header_length and total_length: I have removed those from this PR